app/helpers/application_helper.rb

```diff
@@ -793,7 +793,8 @@ def display_adv_search?
        container_route container_project container_replicator container_image
        container_image_registry persistent_volume container_build
        ems_container vm miq_template offline retired templates
-       host service storage ems_cloud ems_cluster flavor
+       ems_middleware middleware_server middleware_domain middleware_messaging middleware_deployment
```

Comment:
It is not visible from navigation, and it's accessible only by navigating to Domains -> Groups. From what I've seen the filtering option is available only in items accessible from navigation and not nested things.